com.github.Bleuzen.FFaudioConverter.appdata.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.github.Bleuzen.FFaudioConverter</id>
+  <launchable type="desktop-id">ffaudioconverter.desktop</launchable>
+  <project_license>MIT</project_license>
+  <name>FFaudioConverter</name>
+  <summary>Convert audio files</summary>
+  <description>
+    <p>Qt GUI for converting audio files with FFmpeg</p>
+  </description>
+  <categories>
+    <category>Audio</category>
```

Comment:
These go in the `.desktop` file and are auto-extracted.

Reply:
done